velox/vector/ComplexVector.h

```diff
@@ -591,6 +591,12 @@ class MapVector : public ArrayVectorBase {
         type->childAt(1)->toString());
   }
 
+  /// Constructor for creating an empty MapVector for the given rows.
+  MapVector(
```

Comment:
nit: Should something like this exist for arrays and rows ?

Reply:
@kgpai 
Arrays have ARRAY[], which I haven't seen any errors for so far.
Similarly Rows - haven't seen errors either.

Maybe they will fail too if coordinator sends the expression rather than constant blob, like in the case we have discovered.

In any case, these would be separate changes.